src/OrbitGl/BasicPageFaultsTrack.cpp

```diff
@@ -106,4 +106,24 @@ float BasicPageFaultsTrack::GetAnnotatedTrackContentHeight() const {
   return GetGraphContentHeight();
 }
 
+const orbit_client_protos::TimerInfo* BasicPageFaultsTrack::GetLeft(
+    const orbit_client_protos::TimerInfo& /*info*/) const {
+  return nullptr;
+}
+
+const orbit_client_protos::TimerInfo* BasicPageFaultsTrack::GetRight(
+    const orbit_client_protos::TimerInfo& /*info*/) const {
+  return nullptr;
+}
+
+const orbit_client_protos::TimerInfo* BasicPageFaultsTrack::GetUp(
+    const orbit_client_protos::TimerInfo& /*info*/) const {
+  return nullptr;
+}
+
+const orbit_client_protos::TimerInfo* BasicPageFaultsTrack::GetDown(
+    const orbit_client_protos::TimerInfo& /*info*/) const {
+  return nullptr;
+}
+
```

Comment:
In `GraphTrack` we already mention that these are not supported and will always return nullptr. Do we still need to override these methods in the derived classes?

Reply:
right - removed.